atom-publisher-lib/src/main/scala/com/gu/atom/data/DataStore.scala

```diff
@@ -24,7 +24,6 @@ trait DataStore extends DataStoreResult {
   /* this will only allow the update if the version in atom is later
    * than the version stored in the database, otherwise it will report
    * it as a version conflict error */
-  def updateAtom(newAtom: Atom): DataStoreResult[Unit]
 
```

Comment:
Yes, I thought that it would make sense since they both have their own ways of updating atoms.